src/main/java/banking/BankingSystem.java

```diff
@@ -0,0 +1,177 @@
+package banking;
+
+import banking.dao.AccountDAO;
+import banking.model.Account;
+import banking.model.Card;
+import java.util.Scanner;
+import static java.lang.System.exit;
+
+public class BankingSystem {
+
+    private Account currentAccount;
+    private int amountSend;
+    private int choose;
+    private boolean isLogin;
+    private CardGenerator cardGenerator;
+    private CardValidator cardValidator;
+    private AccountDAO accountDAO;
+    private Scanner scanner;
+
+    public BankingSystem(AccountDAO accountDAO) {
+        cardGenerator = new CardGenerator();
+        cardValidator = new CardValidator();
+        this.accountDAO = accountDAO;
+        scanner = new Scanner(System.in);
```

Comment:
The scanner is never closed. Can you please close it?

Reply:
fixed